test/cljam/io/cram/codecs/rans4x8_test.clj

```diff
@@ -0,0 +1,86 @@
+(ns cljam.io.cram.codecs.rans4x8-test
+  (:require [cljam.io.cram.codecs.rans4x8 :as rans]
+            [cljam.io.util.byte-buffer :as bb]
+            [clojure.java.io :as io]
+            [clojure.test :refer [deftest is testing]])
+  (:import [java.io FileInputStream]
+           [java.nio ByteBuffer]
+           [java.util Arrays]))
+
+(deftest read-frequencies0-test
+  ;; cited from https://github.com/samtools/hts-specs/blob/346a94a9980b0105e926a019b4e62fa1b9e30910/CRAMcodecs.tex#L389-L394
+  (let [arr (byte-array [0x61      0x87 0x47
+                         0x62 0x02 0x82 0xe8
+                         #_>       0x81 0x74
```

Comment:
These `#_>` are just for preventing Clojure formatters from "fixing" indentation for these lines.